tests/net/socket/tcp/src/main.c

```diff
@@ -234,15 +234,59 @@ void test_v6_send_recv(void)
 	k_sleep(TCP_TEARDOWN_TIMEOUT);
 }
 
-void test_v4_sendto_recvfrom(void)
+/* Test the stack behavior with a resonable sized block data, be sure to have multiple packets */
+#define TEST_BLOCK_SIZE 60000
+
+static uint8_t testBlockTcp[TEST_BLOCK_SIZE];
+static uint8_t testBlockTcpRx[TEST_BLOCK_SIZE + 10];
```

Comment:
This will greatly increase the RAM requirements of the test suite. Perhaps it'd be better to use smaller buffers (1k for example) and just re-send it several times?

Reply:
Also, please use snake_case for Zephyr code.